src/ui/comment/CommentCardContent.jsx

```diff
@@ -109,6 +109,7 @@ function CommentCardContent({
 		!hasReplyInNonIdleBusyState;
 
 	const resolvedAuthorAndTimestampLabel = useAuthorAndTimestampLabel(reviewAnnotation, true);
+	const [authorLabel, timestampLabel] = resolvedAuthorAndTimestampLabel.split(' – ');
```

Comment:
The useAuthorAndTimestampLabel API is pretty shit, should not have returned a direct string, but just the parts.
Since it's not yet public (documented at least lol), we can change it!

We should move it to this configuration package and directly change it here. Then it will also work for translated labels, that may not contain the ` - ` part. Let me know if you need assistance with that.